tests/tests_lite/plugins/environments/test_slurm_environment.py

```diff
@@ -81,7 +81,17 @@ def test_attributes_from_environment_variables(caplog):
 
 @pytest.mark.parametrize(
     "slurm_node_list,expected",
-    [("alpha,beta,gamma", "alpha"), ("alpha beta gamma", "alpha"), ("1.2.3.[100-110]", "1.2.3.100")],
+    [
+        ("127.0.0.1", "127.0.0.1"),
+        ("alpha", "alpha"),
+        ("alpha,beta,gamma", "alpha"),
+        ("alpha beta gamma", "alpha"),
+        ("1.2.3.[100-110]", "1.2.3.100"),
+        ("1.2.3.[089, 100-110]", "1.2.3.089"),
+        ("host[5-9,1]", "host5"),
+        ("alpha, host[5-9], gamma", "alpha"),
```

Comment:
For this particular case, I'm not sure whether it is meant to be supported, i.e., mixing comma separated list with ranges. Can't find any documentation regarding this on the SLURM side. In any case we support parsing it.